booklets_text/2013.txt

```diff
@@ -1795,13 +1795,13 @@ Primary Technology: Java
 Secondary Technology: XML
 Project Title: WOWit
 Name: Criostóir O'Codlatáin Lachtna
-Email: criostoir.ocodlatainlachtna2@
-mail.dcu.ie
+Email: criostoir.ocodlatainlachtna2@mail.dcu.ie
 Name: Pete Maguire
 Email: peter.maguire22@mail.dcu.ie
 Name: Desmond Brodie
+Email: 
 Programme: Enterprise Computing
-Supervisors: Cathal Gurrin & Alan Smeaton
+Supervisor: Cathal Gurrin & Alan Smeaton
```

Comment:
Single exception in the data where we have Supervisors instead of Supervisor. Just rename the field to fix this (we can have multiple supervisors in the same field for simplicity)